src/api.ts

```diff
@@ -142,6 +142,12 @@ export default class Chromeless<T extends any> implements Promise<T> {
     return this
   }
 
+  setDocumentContent(html: string): Chromeless<T> {
```

Comment:
I renamed `setDocumentContent()` to `setHtml()` in 1f9a300eb59953481d93f0f6c24d4461580df4ba.

My next step is to implement `getHtml()`.